flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/JobInitializationMetricsBuilder.java

```diff
@@ -119,13 +119,23 @@ private static void aggregateMetrics(
         }
     }
 
-    public void reportInitializationMetrics(SubTaskInitializationMetrics initializationMetrics) {
+    public void reportInitializationMetrics(
+            ExecutionAttemptID executionAttemptId,
+            SubTaskInitializationMetrics initializationMetrics) {
         LOG.debug("Reported SubTaskInitializationMetrics={}", initializationMetrics);
```

Comment:
Should we also log the executionAttemptId?
`LOG.debug("Reported SubTaskInitializationMetrics={} for execution attempt {}.", initializationMetrics, executionAttemptId);`